app/src/backend/formulas.ts

```diff
@@ -244,8 +246,7 @@ export async function handleDomesticWasteWaterFormula(
   }
 
   const methaneProductionCapacity = DEFAULT_METHANE_PRODUCTION_CAPACITY; // TODO should this only be handled UI-side?
-  const removedSludge =
-    data["wastewater-inside-domestic-calculator-total-organic-sludge-removed"];
+  const removedSludge = data["total-organic-sludge-removed"];
```

Comment:
I think we might need to revert this change?
And need a better mapping of input IDs to actual formula values that allows the different names from different activities to be read here.

Reply:
this wasn't a change from the manual input. The formula was just referring to the wrong property